### DISCUSSION TITLE
5.5.3 not available on Google Play or F-Droid?

### DISCUSSION THREAD

**Discussion by pulsar256** [daneren2005/Subsonic#1119] [Q&A] [answered]
Hi,

I was trying to understand or read up what the current policy of publishing DSub to stores such as Google Play or F-Droid is. Could not find any information on that yet. The version available on the mentioned app stores is from 2020 (5.5.1) and 5.5.3 has been released a couple of months ago. Are we supposed to download and install the APK manually or is there any other (semi) official way to have it being managed / updated by any app store?

**Comment by popeen** [comment, marked as answer]
Hi

I don't think Scott has addressed this but here are my 2 cents.

F-Droid has the latest release available here
https://f-droid.org/packages/github.daneren2005.dsub/

For Google Play I can only guess why there has been no update but based on my own experience with a fork on Play that guess would be that there are some smaller changes that need to be done before google would accept an update.  Targeting newer android version, removing/rewriting auto detect home wifi (checking wifi name in background is considered [background location](https://support.google.com/googleplay/android-developer/answer/9799150)) etc

**Reply by pulsar256** [reply]
Thanks for clarifying that. Also from the looks of it F-Droid is not able to upgrade packages installed via Google Play (I have memories of this not always being like that) and will show the installed instead of available version. My mistake on that.
